pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -238,10 +247,18 @@ module.exports = {
 
       I.refreshPage();
       I.wait(5);
+
+      if (!added && i === 5) {
```

Comment:
What is this magic number about? In such cases we should have a comment, since you can't infer it from the code.

Reply:
One more: if we break at `i == 5`, why do we need to set ` i < 30`?

Reply:
My explanation: I've added that condition because if I am checking that rule is NOT added I think that I don't need to wait 150 seconds for a response. So I am waiting just 30 seconds for NOT added rule. I think that 30 iterations were added for a really slow response from API. I will add a comment to the code but If you want a better explanation we can call each other. :)

Reply:
Outdated :)